content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -85,73 +85,73 @@ to understand how `X-Forwarded-For` headers and trusted client addresses are det
 
 1. Specify `numTrustedProxies` as 2 either using `MeshConfig` or an `proxy.istop/io/config` annotation. If you are using `MeshConfig`, run the following command to create a file named `topology.yaml` and apply it to your cluster:
 
-    {{< text bash >}}
-    $ cat << EOF > topology.yaml
-      apiVersion: install.istio.io/v1alpha1
-      kind: IstioOperator
-      spec:
-        meshConfig:
-          defaultConfig:
-            gatewayTopology:
-              numTrustedProxies: 2
-      EOF
-    $ istioctl manifest apply -f topology.yaml
-    {{< /text >}}
-
-    {{< idea >}}
-    If you previously installed an Istio ingress gateway, restart all ingress gateway pods after step 1.
-    {{</ idea >}}
+{{< text bash >}}
+$ cat << EOF > topology.yaml
+  apiVersion: install.istio.io/v1alpha1
+  kind: IstioOperator
+  spec:
+    meshConfig:
+      defaultConfig:
+        gatewayTopology:
+          numTrustedProxies: 2
+  EOF
+$ istioctl manifest apply -f topology.yaml
+{{< /text >}}
+
+{{< idea >}}
+If you previously installed an Istio ingress gateway, restart all ingress gateway pods after step 1.
+{{</ idea >}}
```

Comment:
Sorry, I wasn't clear. These need to be indented 4 spaces, or the bullet numbers will reset to 1. What I meant was that I would not indent the YAML 2 extra spaces. 
```suggestion
    {{< text bash >}}
    $ cat << EOF > topology.yaml
    apiVersion: install.istio.io/v1alpha1
    kind: IstioOperator
    spec:
      meshConfig:
        defaultConfig:
          gatewayTopology:
            numTrustedProxies: 2
    EOF
    $ istioctl manifest apply -f topology.yaml
    {{< /text >}}

    {{< idea >}}
    If you previously installed an Istio ingress gateway, restart all ingress gateway pods after step 1.
    {{</ idea >}}
```